portfolio/src/main/webapp/js/comments.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Sends blog-title and user's prefered comment limit to the server
+ * to recieve all relevant comments
+ *
+ * Generates comment html using the recieved json and
+ * a delete button to remove each comment via id
+ */
+function getComments() {
+    const limitInput = document.getElementById('comment-limit');
+    const value = limitInput.options[limitInput.selectedIndex].value;
+  
+    let blogTitle = document.getElementById('blog-title').innerHTML;
+    blogTitle = blogTitle.split(' ').join('_'); // Clean the url
+    let queryString = '/get-comments?' + 'blog-title=' + blogTitle;
+  
+    if (value != 'null') {
+      queryString += '&limit=' + value;
+    }
+  
+    const languageInput = document.getElementById('comment-language');
+    const language = languageInput.options[languageInput.selectedIndex].value;
+    queryString += '&language=' + language;
+  
+  
+    fetch(queryString)
```

Comment:
1. Comment what's going on in this block of code.
2. Consider creating a function that handles the work you do if err is caught.